src/the_program_newloop.cpp

```diff
@@ -244,6 +257,14 @@ int c_the_program_newloop::main_execution() {
 	this->use_options_peerref();
 
 	my_cap::drop_privileges_before_mainloop(); // [security] we do not need special privileges since we enter main loop now
+
+	if (m_argm.count("special-ubsan1"))
```

Comment:
there is/was a bug that UB on signed char is not detected;

how ever this will be used in our integration tests, if our program aborts on UB,
there will be automated test for it possibly later